### ISSUE TITLE
Y Axis in Peak Editor skews chromatogram

### ISSUE THREAD

**Issue by kehj215** [ElucidataInc/ElMaven#1399]
**Describe the bug**
When scrolling through samples in peak editor, some have a skewed, incredibly large Y axis that skews the chromatogram so no peaks (or noise) are visible.

**To Reproduce**
1. Go to bookmarked peak
2. Click on peak editor
3. click through samples, some (I've seen it most often with my "blanks") appear with a skewed Y axis.

**Expected behavior**
Y axis is represented in scientific notation so peaks and noise in the chromatogram are visible.

**Screenshots**
one of a blank with a skewed Y axis and one of a blank with an expected Y axis.

**System (please complete the following information):**
 - OS: macOS big sur
 - Version: v0.12.1-alpha
<img width="819" alt="Screen Shot 2021-04-05 at 10 48 57 AM" src="https://user-images.githubusercontent.com/80782500/113587885-6e9d5000-95fd-11eb-9a7a-440c60c328f6.png">
<img width="819" alt="Screen Shot 2021-04-05 at 10 49 54 AM" src="https://user-images.githubusercontent.com/80782500/113587892-70ffaa00-95fd-11eb-8afd-6eb4f33c6598.png">

**Additional context**

**Comment by saif-el** [comment]
@kehj215 Is it possible that the sample "blank_008" has no signal in that window? Even then, the y-axis values should not be ridiculous like that. Can you try moving the vertical slider (beside the chromatogram plot in peak-editor)? It essentially downscales the y-axis for cases when peaks of interest are too small compared to the highest intensities in the chromatogram.

**Comment by kehj215** [comment]
Yes, in fact I expect there to be no peaks in the blank, but I do anticipate some noise. I tried moving the vertical slider, but it has no effect.

**Comment by saif-el** [comment]
Here too, can we get one of those blank samples where the issue is easily reproduced?
